composer/callbacks/__init__.py

```diff
@@ -24,6 +25,7 @@
     'CheckpointSaver',
     'MLPerfCallback',
     'EarlyStopper',
+    'ExportForInference',
```

Comment:
Could you add this to https://github.com/mosaicml/composer/blob/dev/docs/source/trainer/callbacks.rst as well?

Reply:
Added.